build.xml

```diff
@@ -62,77 +62,77 @@
     <target name="-download-unfolding-maps">
         <mkdir dir="${ivy.lib.dir}"/>
         <echo message="installing unfolding-maps..."/>
-        <get src="https://github.com/constellation-app/third-party-dependencies/raw/master/unfolding-master-20171010.jar"
+        <get src="https://raw.githubusercontent.com/constellation-app/third-party-dependencies/master/unfolding-master-20171010.jar"
```

Comment:
i'd assume you don't need the `raw=true` parameter for all these urls now?

Reply:
Seems to download without it - Just checking through a browser request to that URL

Reply:
yeah seems to be ok probs because the domain is `raw.githubusercontent.com`. This was what GitHub redirected it to so I was just putting that in.